PatientRegistrationApp/src/components/HouseholdReviewSubmit.vue

```diff
@@ -53,12 +72,12 @@
 						<div class="font-weight-medium primary--text">Personal Info: Household Member #1</div>
 						<div class="font-weight-medium">Name:  <span class="font-weight-regular">{{dataHouseholdPersonalInfo[0].householdFamilyName}}, 
 						{{dataHouseholdPersonalInfo[0].householdGivenName}} {{dataHouseholdPersonalInfo[0].householdMiddleName}} {{dataHouseholdPersonalInfo[0].householdSuffix}}</span></div>
-						<div class="font-weight-medium">DOB:  <span class="font-weight-regular">{{dataHouseholdPersonalInfo[0].householdBirthDate}}</span></div>
+						<div class="font-weight-medium">Date of Birth:  <span class="font-weight-regular">{{dataHouseholdPersonalInfo[0].householdBirthDate}}</span></div>
```

Comment:
Are you just cleaning up code here as you go?

Reply:
I could either do the change here or as part of the clean-up card.  This is the change that Todd' requested during the frontend meeting.